app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -245,13 +271,11 @@ export default {
       },
     },
     showSelfAssignBanner() {
-      if (this.message !== '' && !this.isOnPrivateNote) {
+      if (!this.isOnPrivateNote) {
         if (!this.assignedAgent) {
-          return true;
-        }
-        if (this.assignedAgent.id !== this.currentUser.id) {
-          return true;
+          return this.isOnlyAgentReplyMine || this.hasNoAgentReplied;
```

Comment:
@nithindavid Can you please share the logic to show the banner here?

If there is no assigned agent, then we show the banner if no agent has replied to it or if I'm the only one replied to it. So if I'm replied to a conversation that has been assigned to a different person, then this wouldn't be shown right?

In the current scenario, we would show the banner if I'm replying to a conversation which is not assigned to me or unassigned.

Reply:
We could show banner if assigned to another agent but if the replies are only from you or no one has replied. Right now this shows always on all tickets.